### PR TITLE
Webpack: Make bash formatting in install statements consistent

### PR DESCRIPTION
**Pull request by nekosoffy**
## Because
In the whole article, all commands detailing the installing of a package are written with a bash formatting, except for the edited one. When skimming through the page, one can end up skipping the command instruction this way.

## This PR
- Changes how a specific sentence of the page is formatted for better clarity and consistency.

## Issue

Closes #XXXXX

## Pull Request Requirements

-   [x] I have thoroughly read and understand [The Odin Project curriculum contributing guide](https://github.com/TheOdinProject/curriculum/blob/main/CONTRIBUTING.md)
-   [x] The title of this PR follows the `location of change: brief description of change` format, e.g. `Intro to HTML and CSS lesson: Fix link text`
-   [x] The `Because` section summarizes the reason for this PR
-   [x] The `This PR` section has a bullet point list describing the changes in this PR
-   [x] If this PR addresses an open issue, it is linked in the `Issue` section
-   [x] If any lesson files are included in this PR, they have been previewed with the [Markdown preview tool](https://www.theodinproject.com/lessons/preview) to ensure it is formatted correctly
-   [x] If any lesson files are included in this PR, they follow the [Layout Style Guide](https://github.com/TheOdinProject/curriculum/blob/main/LAYOUT_STYLE_GUIDE.md)

### CODE REVIEW

File: javascript/organizing_your_javascript_code/webpack.md [hunk 1]
```diff
@@ -259,7 +259,13 @@ There are three different ways you could be dealing with local image files:
...
 1. **Image files we reference in our HTML template, e.g. as the `src` of an `<img>`**
 
-   We need to install and tell Webpack to use something called `html-loader`, which will detect image file paths in our HTML template and load the right image files for us. Without this, `./odin.png` would just be a bit of text that will no longer reference the correct file once we run Webpack to build into `dist`. We can install it with `npm install --save-dev html-loader`, then add the following object to the `modules.rules` array within `webpack.config.js`:
+   We need to install and tell Webpack to use something called `html-loader`, which will detect image file paths in our HTML template and load the right image files for us. Without this, `./odin.png` would just be a bit of text that will no longer reference the correct file once we run Webpack to build into `dist`. Let's install it:
+
+```bash
+npm install --save-dev html-loader
+```
+
+then add the following object to the `modules.rules` array within `webpack.config.js`:
```
Comment (MaoShizhong) [inline javascript/organizing_your_javascript_code/webpack.md:268]:
````suggestion
   We need to install and tell Webpack to use something called `html-loader`, which will detect image file paths in our HTML template and load the right image files for us. Without this, `./odin.png` would just be a bit of text that will no longer reference the correct file once we run Webpack to build into `dist`. Let's install it:

   ```bash
   npm install --save-dev html-loader
   ```

   Then, add the following object to the `modules.rules` array within `webpack.config.js`:
````

A fair improvement made. Just a nit that needs changing as above to make sure things are aligned properly and the list numbering doesn't get broken up (as per the screenshot below):

image

**Review by MaoShizhong — CHANGES_REQUESTED** [review]

**Comment by MaoShizhong** [comment]
There are a lot of extra unwanted changes that look like it came from a formatter on your local device. Could these changes be reverted please? We only need the changes requested up above.

**Comment by nekosoffy** [comment]
There you go! Are the changes satisfactory?

**Comment by MaoShizhong** [comment]
Just need to remove the spaces from those "blank" lines so that they're actually blank lines, then we should be good to go!

**Comment by nekosoffy** [comment]
Woops, my bad!

**Review by MaoShizhong — APPROVED** [review]
:rocket:

**Pull request merged**